modules.yml

```diff
@@ -1065,3 +1065,13 @@
   path: reporting/xml
   required_keys: []
   version: '1.1'
+- author: Ryan Hays (@_ryanhays)
+  dependencies: []
+  description: Imports hosts and ports into the respective databases from Masscan
+    XML output.
+  files: []
+  last_updated: '2020-04-07'
+  name: Masscan XML Output Importer
+  path: import/masscan
```

Comment:
Please sort the module index by path. If you do a complete index, that's the order that it creates the index. It provides better consistency and clarity for future updates.